onnx/onnx-ml.proto

```diff
@@ -476,6 +476,23 @@ message TypeProto {
     Map map_type = 5;
 
   }
+
+  // An optional denotation can be used to denote the whole 
+  // type with a standard semantic description as to what is 
+  // stored inside
+  optional string denotation = 6;
+}
+
+// A set of pre-defined constants to be used as values for
+// the standard denotation field in Tensor for semantic 
+// description of the tensor.
+message TypeDenotationConstProto {
```

Comment:
Please see my comment above about a generic tensor

Reply:
I was thinking we make denotation optional, and "generic" would be that there is no denotation at all.    Are you saying we should add a TENSOR to the constproto as the default/0 case so that you could provide a tensor denotation and say it is just a "tensor" ?

Reply:
Agreed.